examples/bulk_api_usecases/bulk-operations-csv/bulk_delete_csv.bal

```diff
@@ -18,15 +18,23 @@ import ballerina/log;
 import ballerina/regex;
 import ballerinax/salesforce.bulk;
 import ballerinax/salesforce;
+import ballerina/os;
 
-// Create Salesforce client configuration by reading from config file.
+// Create Salesforce client configuration by reading from environemnt.
+configurable string clientId = os:getEnv("CLIENT_ID");
+configurable string clientSecret = os:getEnv("CLIENT_SECRET");
+configurable string refreshToken = os:getEnv("REFRESH_TOKEN");
+configurable string refreshUrl = os:getEnv("REFRESH_URL");
+configurable string baseUrl = os:getEnv("EP_URL");
+
+// Using direct-token config for client configuration
 salesforce:ConnectionConfig sfConfig = {
-    baseUrl: "<BASE_URL>",
-    clientConfig: {
-        clientId: "<CLIENT_ID>",
-        clientSecret: "<CLIENT_SECRET>",
-        refreshToken: "<REFESH_TOKEN>",
-        refreshUrl: "<REFRESH_URL>"
+    baseUrl: baseUrl,
```

Comment:
We can use
```
ConnectionConfig sfConfig = {
    baseUrl,
    auth: {
        clientId,
        clientSecret,
        refreshToken,
        refreshUrl
    }
};
``` 
as the variables used are same as the parameters. Replace with that approach here and in other places if it's appropriate